lib/puppet/pops/issues.rb

```diff
@@ -377,29 +377,29 @@ def self.hard_issue(issue_code, *args, &block)
   # This is not supported in 3x, but it allowed in 4x.
   #
   UNSUPPORTED_RANGE = issue :UNSUPPORTED_RANGE, :count do
-    _("Attempt to use unsupported range in %{expression}, %{count} values given for max 1") % { expression: label.a_an(semantic), count: count }
+    _("Attempt to use unsupported range in an expression of type %{expression_type}. Was given %{count} where max 1 was accepted.") % { expression_type: label(semantic), count: count }
```

Comment:
```suggestion
    _("Attempt to use unsupported range in an expression of type %{expression_type}. Was given %{count} where max 1 is accepted.") % { expression_type: label(semantic), count: count }
```
Rationale: it *still* only accepts 1. The past tense "1 was accepted" is confusing as it begs the question if it is now accepted (or will/could be).

Reply:
Alternatively, to be more like other messages `"Got %{count} where max 1 is accepted"` as that also gets rid of the past tense referring to the time the error event occurred.

Reply:
Additionally, there is a typo on line 377, missing the word "is" in the second half of the sentence: "This is not supported in 3x, but it is allowed in 4x."

Reply:
thx, @jbondpdx for catching that typo. i'll fix it.